process-engine/bpmn-studio
# 🐛Fix bug where http requests are send to about:open-diagrams

## Changes

1. Fix bug where http requests are send to about:open-diagrams

## Issues

Closes #YourIssueNumber

PR: #NumberOfThisPR

## How to test the changes

- open the inspect view with a remote processengine
- click on a open diagram
- see that there are not a lot of errors and notifications
